x-pack/plugins/saved_objects_tagging/public/management/components/_action_bar.scss

```diff
@@ -0,0 +1,19 @@
+.tagMgt__actionBar {
+  > .euiHorizontalRule {
+    margin-bottom: 0;
+  }
```

Comment:
**Suggested recourse:**

Remove margin from your instance with `<EuiHorizontalRule margin="none" />` and add a `<EuiSpacer size="xs">` before it.

Reply:
https://github.com/elastic/kibana/pull/82816/commits/eebe986daa60c8b76c3cebac19ecc1d11f1cf736